test/services/db-export/table-export.service.test.js

```diff
@@ -13,26 +13,26 @@ const ConvertToCSVService = require('../../../app/services/db-export/convert-to-
 const CompressFilesService = require('../../../app/services/db-export/compress-files.service.js')
 const DeleteFileService = require('../../../app/services/db-export/delete-file.service.js')
 const ExportDataFilesService = require('../../../app/services/db-export/export-data-files.service.js')
-const FetchBillingChargeCategoriesService = require('../../../app/services/db-export/fetch-billing-charge-categories.service.js')
+const FetchTableService = require('../../../app/services/db-export/fetch-table.service.js')
 const SendToS3BucketService = require('../../../app/services/db-export/send-to-s3-bucket.service.js')
 
 // Thing under test
-const DbExportService = require('../../../app/services/db-export/db-export.service.js')
+const TableExportService = require('../../../app/services/db-export/table-export.service.js')
 
 describe('Db Export service', () => {
```

Comment:
Think this top-level describe needs updating

```suggestion
describe('Table export service', () => {
```